R/AllGenerics.R

```diff
@@ -503,7 +520,7 @@ setGeneric("chromPeakData<-", function(object, value)
 #' ## spectra variable *peak_id* contain the row names of the peaks in the
```

Comment:
note: you should also change here *peak_id* to *chrom_peak_id*

Reply:
ah I knew i missed some ! I'll check around to be sure i didn't miss anymore